app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Base class for vaccine errors."""
+    pass

```

Comment:
You've defined a custom exception class `VaccineError` which is a good practice. However, according to the checklist, you should provide a descriptive error message. You can do it by overriding `__init__` method of the `Exception` class. For example:

python
class VaccineError(Exception):
    def __init__(self):
        super().__init__('Vaccine related error occurred.')
```